src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  width: $calendar-width;
+  padding: $day-padding;
+
+  &__day {
+    box-sizing: border-box;
+    background-color: #eee;
+    border: 1px solid #000;
+    justify-content: center;
+    align-items: center;
+    width: 100px;
+    height: 100px;
+    transition-duration: 0.5s;
```

Comment:
The `transition-duration` property is duplicated here and in line 29. You should define it once to avoid redundancy and potential conflicts.